client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -98,9 +98,21 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 									required={ fieldName !== 'state' }
 									value={ this.state[ fieldName ] }
 								/>
+								{ fieldName !== 'state' && (
+									<FormInputValidation
+										isError={ ! this.state[ fieldName ] }
+										text={ translate( 'Required field.' ) }
+									/>
+								) }
 							</FormFieldset>
 						) ) }
-						<Button disabled={ isRequestingSettings } primary type="submit">
+						<Button
+							disabled={
+								isRequestingSettings || some( omit( this.state, 'state' ), val => val === '' )
```

Comment:
`val === ''` is different than the `! this.state[ fieldName ]` check we have for displaying a validation error above. Should we be consistent about how we check whether a field has an error?

Reply:
Do we want to have a separate method for the `some( omit( this.state, 'state' ), val => val === ''` logic for better readability and comprehensibility?